pkg/utils/constants.go

```diff
@@ -57,4 +57,10 @@ const (
 	Softlayer = "softlayer"
 	// SatelliteProvider ...
 	SatelliteProvider = "upi"
+	// VPCGen2 refers to VPC cluster type ...
```

Comment:
@GunaKKIBM in the last review meeting we had some confusion on terms w.r.t to infrastructure vs cluster.
I hope the below constants are for cluster, I mean `vpc-gen2_cruiser` means the cluster is of type vpc and `satellite_cruiser` its satellite cluster irrespective of the infrasture?